### PR TITLE
[CAS-781] Add query banned users docs

### PR DESCRIPTION
**Pull request by adasiewiczr**
### Description
- [Docs](https://getstream.io/chat/docs/react/moderation/?language=kotlin)
- [LLC Cookbook](https://github.com/GetStream/stream-chat-android/wiki/Client-Cookbook#retrieving-banned-users-from-specific-channels)

### Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] PR targets the `develop` branch
- [ ] Changelog updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] Comparison screenshots added for visual changes
- [x] Reviewers added

### CODE REVIEW

File: stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt [hunk 1]
```diff
@@ -1141,6 +1141,7 @@ public class ChatClient internal constructor(
     }
 
     @CheckResult
+    @JvmOverloads
```
Comment (bychkovdmitry) [inline stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt:1144]: I think @JvmOverloads is not very helpful for java users as they still can't use the function with arbitrary parameters. E.g. somebody wants to use only last 2 parameters, still needs to pass something for the rest.

Comment (adasiewiczr) [reply stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt:1144]: True but if you want to use only the first parameter, you don't have to pass others. I think that usually, users are going to use the first few parameters and JvmOverloads is helpful in such cases.

Comment (bychkovdmitry) [reply stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt:1135]: Ok, I'm just worried that only first 1 or 2 overloads are useful, and others will mostly like not match the exact client's needs.

**Review by bychkovdmitry — APPROVED** [review]

File: stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt (hunk 1 above)
Comment (adasiewiczr) [reply stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt:1135]: yeah :( I'll leave it for now but, maybe we should somehow group pagination-related parameters into separate classes 🤔

**Pull request merged**